pyproject.toml

```diff
@@ -72,6 +72,7 @@ doc = [
     "typing_extensions",
     "exceptiongroup",
     "ipython[test]",
+    "toml",
```

Comment:
recommend `tomli ; python_version<'3.11'`

Reply:
So something like this ? 
```suggestion
    "toml",
    "tomli ; python_version<'3.11'"
```

Reply:
yeah, though with that trailing slash, and probably sorted :blush: . 

Then the matching `import tomllib/import tomli as tomllib` try/except block where it's used.